osu.Game/Online/API/APIMod.cs

```diff
@@ -51,7 +52,10 @@ public Mod ToMod(Ruleset ruleset)
             Mod resultMod = ruleset.CreateModFromAcronym(Acronym);
 
             if (resultMod == null)
-                throw new InvalidOperationException($"There is no mod in the ruleset ({ruleset.ShortName}) matching the acronym {Acronym}.");
+            {
+                Logger.Log($"There is no mod in the ruleset ({ruleset.ShortName}) matching the acronym {Acronym}.");
```

Comment:
Is this intentionally logged at the default level rather than at `LogLevel.Important` like in player?

Reply:
Yeah, this was intentional. I was thinking this would be good to know when looking at user logs, but not something we want to flood a user's notifications with.